### PR TITLE
Update archival scripts, styles, add docs link

### PR DESCRIPTION
**Pull request by feedmeapples**
image

### CODE REVIEW

**Comment by cullywakelin** [comment]
> This namespace is currently not enabled for archival.

Consider changing to:

Archival is not enabled for this namespace.

To enable archival for Event Histories set HistoryArchivalState to enabled by running this command:
...

---

You may want to consider removing the Visibility command altogether because that does not even work right now.

I have a revised Archival guide in the works - but there are other things with higher priority right now so learn-archival.md is the best we have that is published.

File: client/routes/namespace/workflow-archival/components/archival-disabled-messaging/constants.js [hunk 1]
```diff
@@ -6,11 +6,11 @@ export const archivalDisabledMessage = [
 export const historyArchivalDisabledMessage =
   'Set HistoryArchivalState to ENABLED by running command:';
```
Comment (cullywakelin) [inline client/routes/namespace/workflow-archival/components/archival-disabled-messaging/constants.js:7]: Consider changing to:
"Run this command to enable Archival for Event Histories:"

File: client/routes/namespace/workflow-archival/components/archival-disabled-messaging/index.vue [hunk 2]
```diff
@@ -16,17 +16,25 @@
...
     <div v-if="!isVisibilityArchivalEnabled" class="message-group">
       <p>
         {{ visibilityArchivalDisabledMessage }}
       </p>
-      <div v-if="visibilityArchivalEnableCommand">
-        <pre>{{ visibilityArchivalEnableCommand }}</pre>
-      </div>
+      <code lang="bash" v-if="visibilityArchivalEnableCommand">{{
+        visibilityArchivalEnableCommand
+      }}</code>
```
Comment (cullywakelin) [inline client/routes/namespace/workflow-archival/components/archival-disabled-messaging/index.vue:29]: You may want to omit any mention of archiving Visibility records - right now those do not even work AFAIK.

The new guide I am writing covers that detail - but the current one does not, and I don't have an ETA of when the new one will be published at this point.

File: client/routes/namespace/workflow-archival/components/archival-disabled-messaging/index.vue [hunk 3]
```diff
@@ -16,17 +16,25 @@
...
+    </div>
+    <div class="message-group">
+      <p class="docs">
+        For more details please check out
+        <a href="https://docs.temporal.io/docs/learn-archival/" target="_blank"
```
Comment (cullywakelin) [inline client/routes/namespace/workflow-archival/components/archival-disabled-messaging/index.vue:34]: Again, hoping to have a new guide at some point soon - but no ETA.

File: client/routes/namespace/workflow-archival/components/archival-disabled-messaging/index.vue (hunk 2 above)
Comment (feedmeapples) [reply client/routes/namespace/workflow-archival/components/archival-disabled-messaging/index.vue:29]: the request on archival page will fail if visibility is disabled. We could provide an option to opt out from having archival view

Comment (cullywakelin) [reply client/routes/namespace/workflow-archival/components/archival-disabled-messaging/index.vue:29]: Perhaps having it there will incentivize the team to get it working ;)

File: client/routes/namespace/workflow-archival/components/archival-disabled-messaging/index.vue [hunk 4]
```diff
@@ -14,19 +14,27 @@
...
+        To enable Visibility Archival:
+      </p>
+      <code lang="bash" v-if="visibilityArchivalEnableCommand">{{
+        visibilityArchivalEnableCommand
+      }}</code>
+    </div>
+    <div class="message-group">
+      <p class="docs">
+        For more details please check out
+        <a href="https://docs.temporal.io/docs/archive-data/" target="_blank"
```
Comment (cullywakelin) [inline client/routes/namespace/workflow-archival/components/archival-disabled-messaging/index.vue:34]: 👍

**Review by cullywakelin — APPROVED** [review]

**Pull request merged**